src/tools/rbd/action/Bench.cc

```diff
@@ -456,7 +491,9 @@ void get_arguments_for_bench(po::options_description *positional,
   add_bench_common_options(positional, options);
 
   options->add_options()
-    ("io-type", po::value<IOType>()->required(), "IO type (read, write, or readwrite(rw))");
+    ("io-type", po::value<IOType>()->required(), "IO type (read, write, or readwrite(rw))")
+    ("pattern-byte", po::value<PatternByte>(),
```

Comment:
This being a write option (i.e. one that doesn't apply in `--io-type read` case), it would probably be better placed in `add_bench_common_options()`, after `rw-mix-read`.   `rbd bench-write` command is still there despite being deprecated...